run-checks

```diff
@@ -144,18 +142,28 @@ run() {
         fi
 }
 
+# Install staticcheck from the correct source so that it
+# will compile and install for the Go version on the host.
+#
+# shellcheck disable=SC2317
+# Disable "command appears to be unreachable" warnings. This
+# function is indirectly called through run(), which confuses
+# shellcheck.
```

Comment:
Usage confuses shellcheck trying to parse this as a directive.

Reply:
Fixed. Great catch!